components/clearbutton/stories/clearbutton.mdx

```diff
@@ -0,0 +1,61 @@
+import { Canvas, ArgTypes, Meta, Description, Title } from '@storybook/blocks';
+
+import * as ClearButton from './clearbutton.stories';
+
+<Meta of={ClearButton} title="Docs" />
+
+<Title of={ClearButton} />
+<Description of={ClearButton} />
+
+### Usage Notes
+
+Use the correct cross icon size that corresponds to the t-shirt size you require for the clear button.
+
+<table>
+  <thead>
+    <tr>
+      <th>**T-Shirt Size**</th>
+      <th>**Icon Size**</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>spectrum-ClearButton--sizeS</td>
+      <td>spectrum-css-icon-Cross75</td>
+    </tr>
+    <tr>
+      <td>spectrum-ClearButton--sizeM</td>
+      <td>spectrum-css-icon-Cross100</td>
+    </tr>
+    <tr>
+      <td>spectrum-ClearButton--sizeL</td>
+      <td>spectrum-css-icon-Cross200</td>
+    </tr>
+    <tr>
+      <td>spectrum-ClearButton--sizeXL</td>
+      <td>spectrum-css-icon-Cross300</td>
+    </tr>
+  </tbody>
+</table>
+
+### Standard
+
+This variant defaults to the `size: m` T-Shirt size.
+
+<Canvas of={ClearButton.Default} />
+
+### Disabled
```

Comment:
I could definitely be convinced that this doesn't need its own story, especially since it's a control toggle, similar to the checkbox.

Reply:
I would keep it. I think we want to show typical and important options and variants on the MDX docs, including some things that may be hidden behind some controls. So a user can get a general overview of what's available.